internal/app/ui/cpwsarea/wsmap/pmap/psettings/screenshot.go

```diff
@@ -38,6 +40,8 @@ func (p *Panel) showScreenshot() {
 		imgui.SetNextItemWidth(-1)
 		imgui.InputText("##screenshot_dir", &cfg.ScreenshotDir)
 
+		imgui.Checkbox("Screenshot in Selection", &p.sessionScreenshot.inselection)
```

Comment:
Actually, I think you can force user to use the grab tool here

```go
if imgui.Checkbox("Screenshot in Selection", &p.sessionScreenshot.inselection) {
    tools.SetSelected(tools.TNGrab)
}
```

Reply:
addressed

Reply:
addressed